src/main/java/com/capitalone/dashboard/logging/LoggingFilter.java

```diff
@@ -52,6 +52,8 @@
 @Order(1)
 public class LoggingFilter implements Filter {
 
+    public static final String IGNORE_TEST_RESULTS_REQUESTS = "/api/quality/test-result";
```

Comment:
@danielyhuang  can we configure this value from properties, that way it will not be static

Reply:
completed as suggested.